packages/api/src/pins.js

```diff
@@ -192,7 +192,7 @@ export async function pinsGet (request, env, ctx) {
  */
 function getPinStatus (pinRequest) {
```

Comment:
Nit: I'd have named this something like `toPinStatusResponse` - this isn't _retrieving_ anything it's transforming something.

Reply:
Agree, I'll change it.